### PR TITLE
Numpad keys will now be recognised by the keybinds menu

### PR DESCRIPTION
**Pull request by GoldenAlpharex**
## About The Pull Request
So, I was trying to rebind some emotes using CTRL+Numpad1, and then noticed it, in fact, did not let me do that. Curious, I reported it and nothing was ever done about it, because the code checks out, right?

Well, ladies and gentlemen, let me show you how great IE is:
image

That's right, all the KeyboardEvent values are undefined, which means that we literally can't use those things. Great.
So I had to make a constant for the numpad, we can easily add more in the future if it's needed, but... Yeah, this was annoying to figure out.

## Why It's Good For The Game
You can finally bind some things to the numpad keys, as intended.
Which means you can do this:
image

## Changelog

:cl: GoldenAlpharex
fix: Numpad keys are now properly recognized by the keybinds menu.
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/PreferencesMenu/KeybindingsPage.tsx [hunk 1]
```diff
@@ -49,6 +49,12 @@ const KEY_CODE_TO_BYOND: Record<string, string> = {
   "UP": "North",
 };
 
+/* So, as it turns out, KeyboardEvent seems to be broken with IE 11, the
+ * DOM_KEY_LOCATION_X codes are all undefined. See this to see why it's 3:
+ * https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/location
+ */
+const DOM_KEY_LOCATION_NUMPAD: number = 3;
```
Comment (stylemistake) [inline tgui/packages/tgui/interfaces/PreferencesMenu/KeybindingsPage.tsx:56]: :number is redundant.

File: tgui/packages/tgui/interfaces/PreferencesMenu/KeybindingsPage.tsx [hunk 2]
```diff
@@ -49,6 +49,12 @@ const KEY_CODE_TO_BYOND: Record<string, string> = {
   "UP": "North",
 };
 
+/* So, as it turns out, KeyboardEvent seems to be broken with IE 11, the
```
Comment (stylemistake) [inline tgui/packages/tgui/interfaces/PreferencesMenu/KeybindingsPage.tsx:52]:
```suggestion
/**
  * So, as it turns out, KeyboardEvent seems to be broken with IE 11, the
```

**Comment by Mothblocks** [comment]
Please mark this as fixing that one issue, it's in the prefs-menu milestone.

**Comment by Mothblocks** [comment]
Fixes #62959

**Comment by GoldenAlpharex** [comment]
> Please mark this as fixing that one issue, it's in the prefs-menu milestone.

Oops I totally missed that, my bad.

**Pull request merged**